src/System.Net.HttpListener/tests/HttpResponseStreamTests.cs

```diff
@@ -403,6 +403,7 @@ public void Dispose()
         [InlineData(true)]
         [InlineData(false)]
         [ActiveIssue(18128, platforms: TestPlatforms.AnyUnix)] // No exception thrown
+        [ActiveIssue(18188, platforms: TestPlatforms.Windows)] // Indeterminate failure - socket not always fully disconnected.        
```

Comment:
https://github.com/dotnet/corefx/issues/18188 is about an intermittent failure on Windows.